packages/mds-agency/request-handlers.ts

```diff
@@ -137,39 +149,37 @@ export const getVehiclesByProvider = async (req: AgencyApiRequest, res: AgencyAp
 
   try {
     const response = await getVehicles(skip, take, url, req.query, provider_id)
-    return res.status(200).send(response)
+    const { version } = res.locals
+    return res.status(200).send({ version, ...response })
   } catch (err) {
     logger.error('getVehicles fail', err)
-    return res.status(500).send(new ServerError())
+    return res.status(500).send(agencyServerError)
   }
 }
 
 export async function updateVehicleFail(
   req: AgencyApiRequest,
-  res: AgencyApiResponse,
+  res: AgencyUpdateVehicleResponse,
   provider_id: UUID,
   device_id: UUID,
   err: Error | string
 ) {
   if (String(err).includes('not found')) {
-    res.status(404).send({
-      error: 'not_found'
-    })
+    res.status(404).send({})
```

Comment:
I think we can just do `res.status(404).send()` without defining an empty object

Reply:
We could, but then the middleware drops `charset utf-8` from the version header, which then makes it slightly different from the other endpoints.